django_rq/queues.py

```diff
@@ -197,12 +197,12 @@ def get_queues(*queue_names, **kwargs):
 
     queue_params = QUEUES[queue_names[0]]
     connection_params = filter_connection_params(queue_params)
-    ret = [get_queue(queue_names[0], **kwargs)]
+    queues = [get_queue(queue_names[0], **kwargs)]
 
     # do consistency checks while building return list
     for name in queue_names[1:]:
         queue = get_queue(name, **kwargs)
-        if queue.__class__ is not ret[0].__class__:
+        if queue.__class__ is not queues[0].__class__:
```

Comment:
Can we change this to use `isinstance`? I try to avoid using `__` properties in my code base as much as possible. `isinstance` also takes class inheritance into account.

Reply:
As I said, `isinstance` is exactly what we **don't** want here. `get_queues` is mostly used for worker initialization and worker must get one queue class. If we want to run worker for two queues where one is of `DjangoRQ` class and another is some `CustomQueue` extending `DjangoRQ` we cannot guarantee that `CustomQueue` has the same implementation for functionality used by worker class.

Regarding `__class__`, I can change that to `type(queue)`. Would that be better?